### DISCUSSION TITLE
Results By Last Modified Time

### DISCUSSION THREAD

**Discussion by joshmedeski** [scambier/obsidian-omnisearch#401] [Ideas]
I would LOVE to see Omnisearch be able to sort results with the last modified time. I would make this a very high weight as what I want to look at the most is what I have most recently touched in my vault.

I'm also a developer and might be able to help with something like this if you're open to it, thanks!

**Comment by scambier** [comment]
Since today's version ([1.26.0](https://github.com/scambier/obsidian-omnisearch/releases/tag/1.26.0)), there is an opt-in recency boost available in settings. In short, it's a [simple formula](https://github.com/scambier/obsidian-omnisearch/blob/89786b712e1a20749968682ad9135e2e172655f7/src/search/search-engine.ts#L179-L197) that multiplies the score of a result (up to a maximum of ~x1.9) depending on its last modified time, and the chosen cutoff.

I'm not sure how this boost will work on the various vaults and settings, since there is already many variables that alter the score of a result, so I encourage interested users to give their feedback 👍

**Reply by ovgolovin** [reply]
Will it be hard to control this setting from **Results weighting** settings? As well as allow using dates from frontmatter properties?

![image](https://github.com/user-attachments/assets/025545ec-e385-4351-a033-58eb236747f3)

**Reply by scambier** [reply]
Request formalized in #469
